modules/integration_aws-elasticsearch/conf/08-cluster-cpu.yaml

```diff
@@ -0,0 +1,24 @@
+module: AWS Elasticsearch
+name: "Cluster CPU"
+
+transformation: ".min(over='45m')"
+aggregation: ""
+filtering: "filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*')"
+
+signals:
+  data_node_cpu:
+    metric: CPUUtilization
+  warm_node_cpu:
+    metric: WarmCPUUtilization
+  signal:
+    formula: union(data_node_cpu, warm_node_cpu)
```

Comment:
We should probably process those signals independently as Ultrawarm nodes do not have the same profile as normal nodes.

Reply:
It's a bit too much of a change for this PR, maybe for another one